models/user.js

```diff
@@ -32,6 +32,7 @@ module.exports = function User(ldapUserArray) {
                 userEntry.mail.map(function(mail) {
```

Comment:
Note pour plus tard:

Je pense que c'est pas la bonne utilisation map, je que ca pourrait etre remplace par:
```suggestion
                userEntry.mail.forEach(function(mail) {
```

ou mieux remplacer le tout par:
```js
userModel.emails = [...new Set(userEntry.mail)];
```

Reply:
Suis aussi pour le spread operator

Reply:
spread, mais sur `userEntry.mail` (et pas l'autre) en etant sur que userEntry existe...